src/users/dto/create-user.dto.ts

```diff
@@ -13,4 +20,12 @@ export class CreateUserDto {
   @IsNotEmpty()
   @MinLength(6)
   password: string;
+
+  @IsOptional()
+  @IsBoolean()
+  isEmailConfirmed: boolean = false;
+
+  @IsOptional()
+  @IsString()
+  emailConfirmationToken: string;
```

Comment:
You don't have token while creating user. It should be send with another request